### DISCUSSION TITLE
File integrity checking possibilities

### DISCUSSION THREAD

**Discussion by Thomas-Mart** [seaweedfs/seaweedfs#4372] [General]
Hi,

I've a question about the file integrity checkin done by seaweedfs. As I think MD5 algorithm is used to check the file integrity after upload. Is there any possibility to use another hashing algorithm e.g. like SHA256?

**Comment by chrislusf** [comment]
CRC and MD5 are used in different places in SeaweedFS. Not designed to support other algorithms.
